test/testcafe/spec-en-leaks/EnglishLeaks_spec.js

```diff
@@ -34,7 +34,9 @@ const parseMockData = () => {
   console.log('================= Parsing mocks for en leaks automation =============');
   fs.readdirSync(mocksFolder).forEach(file => {
     const isIgnored = ignoredMocks.includes(file);
-    if (!isIgnored) {
+    //only allow json mock files
+    const isJsonMock = path.extname(file) == '.json';
```

Comment:
nit: Let's use strict equality please

```suggestion
    const isJsonMock = path.extname(file) === '.json';
```

Reply:
Dang, I was late to the party.

Reply:
whoops, fixed

Reply:
I am glad, bacon failed lint for this